### DISCUSSION TITLE
Landing page for bounties?

### DISCUSSION THREAD

**Discussion by ElliotFriend** [kalepail/stellar-quest-bounties#82] [Bounty Ideas]
Just noticed that https://quest.stellar.org/bounties was added to the gh repo, but it redirects here. Would it be a useful bounty for making a landing page to live there?

Being a real-life SDF web property, though, that might already be in the works and/or under someone else's jurisdiction

**Comment by 3hundred8** [comment]
What I’ve already mentioned earlier is my main idea so far. If it’s all about generating hype and aiming towards a huge group of participants, there should be a certain wow effect. 

Having a countdown on the landing page that doesn’t give away the whole idea at once, but maybe just some hints towards the upcoming projects. 

- A countdown that is running down until the next drop would be a great eye catcher. 

- Keeping the design space like and somehow mysterious. Maybe even sticking to the planet design from the last Badges. 

- an animated universe (like Google maps just in space) with dark planets, once the bounty is published they could be „enlightened“ and by clicking on them you’d get to the bounty itself. 

- sharing links for all social Networks and thumbnails that show the countdown 

Gonna think a bit further about it, need a cool down from work first  😂

**Comment by kalepail** [comment]
I love this idea. I'm 100% down for this bounty to exist. Make it crazy! Make it wild! Maybe this could be our first team bounty! Get both frontend and a designer or two into the mix. Love it!

**Comment by ElliotFriend** [comment]
I'm gonna start trying to write this bounty up. I'm thinking of making it explicit that it **must** be a team bounty (maybe even just for the purpose of practicing that flow for bounties). Any thoughts on that?

**Comment by raeesnazeer** [comment]
I guess we need to get this bounty up, to promote stellar bounties and get more people here.
